include/nbl/video/alloc/SubAllocatedDescriptorSet.h

```diff
@@ -29,13 +29,19 @@ class SubAllocatedDescriptorSet : public core::IReferenceCounted
 		std::shared_ptr<ReservedAllocator> reservedAllocator;
 		size_t reservedSize;
 	};
-	std::unordered_map<uint32_t, SubAllocDescriptorSetRange> m_allocatableRanges = {};
+	std::map<uint32_t, SubAllocDescriptorSetRange> m_allocatableRanges = {};
 
+	#ifdef _NBL_DEBUG
+	std::recursive_mutex stAccessVerfier;
+	#endif // _NBL_DEBUG
+
+	constexpr static inline uint32_t MaxDescriptorSetAllocationAlignment = 64u*1024u; // if you need larger alignments then you're not right in the head
+	constexpr static inline uint32_t MinDescriptorSetAllocationSize = 1u;
```

Comment:
wuut, you don't need this though....? just use 1 for both, Pool Address allocator really doesn't care